.cog/schemas/composable/athena/dataquery.cue

```diff
@@ -0,0 +1,31 @@
+package athena
+
+import (
+	"github.com/grafana/grafana/packages/grafana-schema/src/common"
+)
+
+// Manually converted from https://github.com/grafana/athena-datasource/blob/57ad707147b7a11e9a521a836d6bf9799877e0e3/src/types.ts
+Dataquery: {
+  common.DataQuery
+
+	format: #FormatOptions
+	connectionArgs: #ConnectionArgs
+	table?: string
+	column?: string
+
+	queryID?: string
+
+	rawSQL: string | *""
+}
+
+defaultKey: "__default"
+
+#ConnectionArgs: {
+	region?: string | *"__default"
```

Comment:
AFAIK you can use `region?: string | *defaultKey`

Reply:
Fair point. Somehow I assumed our cue parser wouldn't handle that correctly :sweat_smile: